auto-notify.plugin.zsh

```diff
@@ -57,7 +57,7 @@ function _auto_notify_send() {
 }
 
 function _auto_notify_track() {
-    AUTO_COMMAND="$1"
+    AUTO_COMMAND="${1:-$2}"
```

Comment:
could you document in a code comemnt why we need this fallback?

Reply:
Of course